scripts/select_appointments.sql

```diff
@@ -1 +1,23 @@
 -- retrieve appointment information
+SELECT d.family_name AS doctor,
+       p.family_name, p.given_name,
+       a.start_at, a.length
```

Comment:
This will cause an error since the columns `start_at` and `length` don't exist on the `appointments` table.